src/cli/commands/create/app.ts

```diff
@@ -151,6 +151,7 @@ export async function runBootstrapper(
 
   if (askDatabase.database) {
     deps.push(`nexus-plugin-prisma@${getPrismaPluginVersion()}`)
+    devDeps.push(`prisma2@2.0.0-preview023`)
```

Comment:
This shouldn't be in core. We should create an issue about how its going to be removed.

Reply:
PR created https://github.com/graphql-nexus/nexus-future/issues/553 (very rough 🙈)